README.md

```diff
@@ -32,6 +32,14 @@ $ run.sh
 
 Check out the [contribution guidelines](/CONTRIBUTING.md) for more information about how to contribute to this project.
 
+## Contributors
+
+[0x5c](https://github.com/0x5c)
+
+[bruceyang1998](https://github.com/bruceyang1998)
+
+[madicetea](https://github.com/madicetea)
+
```

Comment:
```suggestion

```

This should not be included in this PR. An issue should be made and the change discussed before it gets potentially implemented.

Reply:
Got it. I'll separate it as its own issue first.